haskell-awk.cabal

```diff
@@ -38,7 +38,7 @@ library
       runtime
   ghc-options: -Wall
   build-depends:
-      base >=4.9.1.0 && <4.12.0.0
+      base >=4.9.1.0 && <5
```

Comment:
as it says at the top, this file is generated by `package.yaml`, so I would have expected to see a corresponding change to that file

Reply:
Oops, I have fixed this easy part.